.github/workflows/it-test.yaml

```diff
@@ -59,6 +59,10 @@ jobs:
           mode: "services"
           profiles: "app,identity"
           aws-account-id: ${{ secrets.STAGING_AWS_ACCOUNT_ID }}
+      - name: Run PHP migrations
```

Comment:
@nicklasmoeller tagging you mainly to ask, is there a better/more optimal way to execute migrations and seed the database for the PHP container? or what I did here is a good way?

Reply:
This is fine - the fresh part should not be needed though.

Did it ever auto-migrate? That sounds incorrect, but I may be wrong. Do you have a workflow run where that was the case?

Reply:
Yes, I do here, https://github.com/monta-app/service-api-gateways/actions/runs/11349905016/job/31620991210?pr=2245

Just click on the Latest#6 drop down you should be able to see previous runs

- initially it was failing because the connectors table was no there.
- i tried to add the migrations step and then it failed because the country 194 did not exist.

I dont know how it used to work in the past to be honest, it somehow seemed to work 😅 it started to fail on [07.10](https://monta-app01.slack.com/archives/C04HQ96GZ8W/p1728310678668839)

Reply:
Ahhhh yes - it's pulling the GH actions from the server repository. The migration was moved out of that step, since we changed how the integration tests are running. Happened in this commit: https://github.com/monta-app/server/commit/9205f3d9f3e31d798c90e48c2c38bd9a5247c321

Migrations are still running in the integration tests in PHP, it's just orchestrated by "paratest" which spins up multiple processes and multiple databases.

Good find, and sorry for the inconvenience 🙏

Reply:
cc @vistik just in case we have other plans on changes to the setup (probably not), but let's keep in mind that it's actually read by this repository, and not a private file 😅

Reply:
ah wow, now I understand how it works, I searching for this god damn GH actions on the marketplace and I could not find it 😅 thanks for explaining it

Reply:
Hahaha, also didn't know that you were using it 😂 Guess we all learned something!